common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/WorkType.scala

```diff
@@ -155,7 +155,7 @@ object WorkType extends Enum[WorkType] {
 
   case object EManuscriptsAsian extends Linked {
     override val id: String = "x"
-    override val label: String = "E-manuscripts, Asian"
+    override val label: String = "Manuscripts"
```

Comment:
Is it worth leaving this here - and maybe leaving a note above saying what the Sierra label is?

Reply:
The Sierra label is captured in the name of the object imo